doc/backing-store-crd.md

```diff
@@ -160,28 +169,27 @@ metadata:
   finalizers:
   - noobaa.io/finalizer
   labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+    app: <>
+  name: <>
+  namespace: <>
 spec:
   pvPool:
-    numVolumes: 3
```

Comment:
What I liked in the examples was that it was clear how the settings in the example are in the YANL, for example after using the flag `--num-volumes 3` we can see the line `numVolumes: 3`.

Reply:
What is wrong to be a source of copy-paste examples?

Reply:
I just think examples are easier to understand...

Reply:
I agree that examples can really help sometimes, maybe instead of removing them completely, could you add an examples section at the end of the file? or even in a different "examples" doc like other source code projects have

Reply:
I too think that examples are quite helpful for beginners (as well devs) to understand and provides much clarity.  

> or even in a different "examples" doc like other source code projects have

Even this could be good approach. A separate directory for all example files. I can help or contribute on this effort if required.

Reply:
I too think that examples are quite helpful for beginners (as well devs) to understand and provides much clarity.

> or even in a different "examples" doc like other source code projects have

Even this could be good approach. A separate directory for all example files. I can help or contribute on this effort if required.